src/server_manager/electron_app/notarize.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 The Outline Authors
```

Comment:
Not Typescript?

Reply:
Since this is a script that is unlikely to change frequently, I don't see an advantage in either dynamically building it on release or checking in both TS and JS files.

Reply:
Maybe add a comment clarifying that this code is not actually part of the app and is only used by the build process?

We could also move it to another directory, like `src/server_manager/build` or just `src/server_manager/`, since there's no app code there and it's where the package.json referring to the file is